[2023-01-04 07:51:29,171] {processor.py:153} INFO - Started process (PID=23191) to work on /home/gustavo/Documents/etl_api_twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04 07:51:29,181] {processor.py:641} INFO - Processing file /home/gustavo/Documents/etl_api_twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04 07:51:29,182] {logging_mixin.py:115} INFO - [2023-01-04 07:51:29,182] {dagbag.py:507} INFO - Filling up the DagBag from /home/gustavo/Documents/etl_api_twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04 07:51:29,220] {logging_mixin.py:115} INFO - [2023-01-04 07:51:29,215] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/gustavo/Documents/etl_api_twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04 07:51:29,220] {logging_mixin.py:115} INFO - [2023-01-04 07:51:29,220] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04 07:51:29,221] {processor.py:653} WARNING - No viable dags retrieved from /home/gustavo/Documents/etl_api_twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04 07:51:29,239] {processor.py:161} INFO - Processing /home/gustavo/Documents/etl_api_twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.073 seconds
[2023-01-04 07:53:20,448] {processor.py:153} INFO - Started process (PID=24263) to work on /home/gustavo/Documents/etl_api_twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04 07:53:20,450] {processor.py:641} INFO - Processing file /home/gustavo/Documents/etl_api_twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04 07:53:20,452] {logging_mixin.py:115} INFO - [2023-01-04 07:53:20,452] {dagbag.py:507} INFO - Filling up the DagBag from /home/gustavo/Documents/etl_api_twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04 07:53:20,519] {logging_mixin.py:115} INFO - [2023-01-04 07:53:20,516] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/gustavo/Documents/etl_api_twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04 07:53:20,519] {logging_mixin.py:115} INFO - [2023-01-04 07:53:20,519] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04 07:53:20,519] {processor.py:653} WARNING - No viable dags retrieved from /home/gustavo/Documents/etl_api_twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04 07:53:20,548] {processor.py:161} INFO - Processing /home/gustavo/Documents/etl_api_twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.111 seconds
